src/Uno.UI.Runtime.Skia.MacOS/MacSkiaHost.cs

```diff
@@ -58,6 +58,12 @@ protected override void Initialize()
 			return;
 		}
 
+		// if packaged as an app bundle, set the resource path base to the Resources folder
+		if (NativeUno.uno_application_is_bundled())
```

Comment:
Let's add a validation for this in the template tests

Reply:
@jeromelaban https://github.com/unoplatform/uno.sdk.extras/pull/78 adds some manual tests that covers font bundling. IMO we should complete code signing testing (on CI) before trying to automate those tests.